src/GOWIN.py

```diff
@@ -9,21 +9,22 @@
 # Default to env if there
 ENV_TOOL_PATH = GET_TOOL_PATH(TOOL_EXE)
 if ENV_TOOL_PATH:
-    GOWIN_PATH = os.path.abspath(os.path.dirname(ENV_TOOL_PATH))
+    GOWIN_PATH = ENV_TOOL_PATH
 else:
-    GOWIN_PATH = "/todo/install/gowin/tools"
+    GOWIN_PATH = "/usr/local/share/gowin/IDE/bin/gw_sh"
 
 class ParsedTimingReport:
     def __init__(self, syn_output):
-        # Split report into individual paths
-        path_tok = "						Path"
-        # Only look at first path for now
-        path_texts = syn_output.split(path_tok)[1:2]
-        # Parse each path
         self.path_reports = {}
-        for path_text in path_texts:
-            path_report = PathReport(path_text)
-            self.path_reports[path_report.path_group] = path_report
+        PATH_REPORT_BEGIN = '''3.3.1 Setup Analysis Report'''
+        PATH_REPORT_END = '''3.3.2 Hold Analysis Report'''
+        syn_report = syn_output.split(PATH_REPORT_BEGIN)[-1].split(PATH_REPORT_END)[0]
+        PATH_SPLIT = "\t\t\t\t\t\tPath"
+        maybe_path_texts = syn_report.split(PATH_SPLIT)
+        for path_text in maybe_path_texts:
+            if "Data Arrival Path:" in path_text:
+                path_report = PathReport(path_text)
+                self.path_reports[path_report.path_group] = path_report
```

Comment:
Something to note here: 
It is expected that `path_report.path_group` is essentially the clock name / clock group / clock domain name. And thus `self.path_reports[path_report.path_group]` is meant to store 'the worst path for that clock domain'. 

So want to double check that if multiple paths exists in the output for the same clock domain that only the first/worst one is saved in the path reports dict

Reply:
From what I gather this seems to be collecting everything correctly - each unit in the report is related to a single clock group AFAICS